src/main/java/frc/robot/subsystems/IntakeSubsystem.java

```diff
@@ -0,0 +1,57 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2017-2018 FIRST. All Rights Reserved.                        */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package frc.robot.subsystems;
+
+import static frc.robot.Constants.*;
+
+import edu.wpi.first.wpilibj.DigitalInput;
+import edu.wpi.first.wpilibj.Spark;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.RobotContainer;
+import frc.robot.commands.IntakeCommand;
+
+public class IntakeSubsystem extends SubsystemBase {
+  // Put methods for controlling this subsystem
```

Comment:
if you wanna have this in, it's cool, but port over these PWM motors to CAN motors with the IDs already in the latest version of temp put in

Reply:
be more specific. What is "this"? the whole block or a specific line?